mifosng-android/build.gradle

```diff
@@ -156,12 +156,12 @@ dependencies {
 
     //DBFlow dependencies
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    annotationProcessor "com.github.Raizlabs.DBFlow.dbflow:dbflow-processor:3.1.1"
-    annotationProcessor "com.github.Raizlabs.DBFlow:dbflow-processor:3.1.1"
+    annotationProcessor "com.github.raizlabs.dbflow.dbflow:dbflow-processor:3.1.1"
+    annotationProcessor "com.github.raizlabs.dbflow:dbflow-processor:3.1.1"
     //annotationProcessor "com.github.Raizlabs.DBFlow.dbflow-core:dbflow-processor:$rootProject.raizLabsDBFlow"
     //implementation "com.github.Raizlabs.DBFlow:dbflow-core:$rootProject.raizLabsDBFlow"
-    implementation "com.github.Raizlabs.DBFlow.dbflow:dbflow:$rootProject.raizLabsDBFlow"
-    kapt "com.github.Raizlabs.DBFlow:dbflow-processor:3.1.1"
+    implementation "com.github.raizlabs.dbflow.dbflow:dbflow:$rootProject.raizLabsDBFlow"
+    kapt "com.github.raizlabs.dbflow:dbflow-processor:3.1.1"
```

Comment:
can you provide a reference for the dbFlow dependency name update?

Reply:
This worked well I had the same issue.